pkg/appender/store.go

```diff
@@ -158,22 +161,30 @@ func (cs *chunkStore) processGetResp(mc *MetricsCache, metric *MetricState, resp
 	cs.chunks[0].appender = app
 	cs.chunks[0].state |= chunkStateFirst
 
-	if resp.Error != nil {
-		// assume the item not found   TODO: check error code
-
-		if metric.newName {
-			path := mc.cfg.TablePath + "/names/" + metric.name
-			putInput := v3io.PutItemInput{Path: path, Attributes: map[string]interface{}{}}
-
-			request, err := mc.container.PutItem(&putInput, metric, mc.nameUpdateChan)
-			if err != nil {
-				// Count errors
-				cs.performanceReporter.IncrementCounter("PutNameError", 1)
+	latencyNano := time.Now().UnixNano() - resp.Request().SendTimeNanoseconds
+	cs.performanceReporter.UpdateHistogram("UpdateMetricLatencyHistogram", latencyNano)
+	methodLogger := mc.logger.GetChild("processGetResp")
```

Comment:
I'm not sure allocating a new logger per call is a good idea. It's also not the practice we have, which is to allocate it with the struct.

Reply:
This is our logging infra. I can just reduce amount of debug logs or stop using child loggers at all

Reply:
Again, the practice is to put the derived logger in the receiver (the struct), not recreating it per method invocation. If you want to specify the method, you can use methods such as `InfoWith` to log and include the method as a variable, e.g. `InfoWith("message here", "method", "processGetResp")`.

If you decide we don't really need these logs, that's ok also.